glib/pkg.yaml

```diff
@@ -25,6 +33,9 @@ steps:
     install:
       - |
         DESTDIR=/rootfs ninja -C _build install
+
+        # we only need the headers and libs
+        rm -rf /rootfs/usr/{bin,share}
```

Comment:
No options to disable mans and utilities?

Reply:
nope, glib and qemu are huge, even if we're just building part of it